package/docker/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM alpine:3.18.3
 
+RUN apk update && \
```

Comment:
@deepthi-mantena Seems like the base image alpine has got fixes for the vulnerabilities and is released. https://hub.docker.com/_/alpine/tags. Instead of updating individual packages, it always better to use upgraded version of the base image wherever possible.

Reply:
@mohan-13 updated base image version and tested it again now there are no vulnerabilities